ui/src/labels/components/LabelList.tsx

```diff
@@ -56,22 +56,25 @@ export default class LabelList extends PureComponent<Props, State> {
     const {sortKey, sortDirection, onClickColumn} = this.props
     return (
       <>
-        <IndexList>
-          <IndexList.Header>
-            <IndexList.HeaderCell
+        <ResourceList>
+          <ResourceList.Header>
+            <ResourceList.Sorter
+              name="Name"
```

Comment:
could this.headerKeys[0] be used here? rather than the string "Name"?

Reply:
Yes, it can. I updated it!